korman/properties/modifiers/anim.py

```diff
@@ -133,6 +133,9 @@ def key_name(self):
 
     def post_export(self, exporter, bo, so):
         # If this object has a physical, we need to tell the simulation iface that it can be animated
+        self.make_physical_movable(so)
+    
+    def make_physical_movable(self, so):
```

Comment:
One last nitpick :tongue: 

This is a private method, so it should be prefixed by an underscore. Also, everything is sorted by alphabetical order in Korman, so this method should be between `export` and `post_export`